.travis.yml

```diff
@@ -1,4 +1,9 @@
 language: php
+
+cache:
+  directories:
+    - $HOME/.composer/cache
```

Comment:
Why not `$HOME/.composer`? And then `hirak/prestissimo` in `before_install`.

Reply:
I wouldn't complicate it with that, cache is enough.